pkg/pillar/cmd/domainmgr/rkt.go

```diff
@@ -0,0 +1,33 @@
+package domainmgr
+
+import (
+	"os"
+
+	docker2aci "github.com/appc/docker2aci/lib"
+	"github.com/appc/docker2aci/lib/common"
+	"github.com/appc/docker2aci/pkg/log"
+)
+
+// rktConvertTarToAci convert an OCI image tarfile into an ACI bundle
+// with thanks to https://github.com/appc/docker2aci
+// from is the file to convert from; to is the path where to place the aci file
+func rktConvertTarToAci(from, to string) ([]string, error) {
+	debug := log.NewNopLogger()
+
+	info := log.NewStdLogger(os.Stderr)
+
+	cfg := docker2aci.CommonConfig{
+		Squash:      true,
+		OutputDir:   to,
+		TmpDir:      os.TempDir(),
```

Comment:
How are we managing the space in /tmp (which is what I assume os.TempDir() uses)?
In general will be keep both the ACI and OCI files while the container is active? If so it might make sense to put both of them in a place we can observe in /persist

Reply:
Not quite. he `TmpDir` is just scratch space for `docker2aci` to use while doing a conversion. When it is done converting, it cleans up the temp dir. The actual output files - the tar file in our standard `ImageID.sha` naming, and the `.aci` file - are both in the standard verified dir, so `/persist/downloads/appImg.obj/verified/<ID>/`.

I was tempted to delete the tar file after the ACI conversion is successful, but unless there is a real issue, I would rather keep it for two reasons:

* aci is heading out with rkt, once runx is in place, and the OCI format will be used
* if we have any issues (functional or security), the trail and existence of files will be helpful

Reply:
OK. As long as TempDir is cleaned up even if there are errors in the conversion.

Reply:
Yup. See [here](https://github.com/appc/docker2aci/blob/248258bd708afc51c1aa0f9e8b826c50d1ce66a8/lib/internal/backend/file/file.go#L103-L107). The first thing it does is `defer os.RemoveAll(tmpdir)`